credits.txt

```diff
@@ -225,31 +238,35 @@ contributed to Endless Sky:
   MCOfficer
   MessyMix
   michel-slm
-  MillerNerd
```

Comment:
This person has contributed, #2006. I don't know why Github doesn't list them.

Reply:
Is it possible that any of these names of people who no longer exist are changed names and thus included in as one of the new names in the list?

Reply:
A few of them, yes. But this one isn't (check the PR author).